jvm/src/main/scala/com/intel/oap/execution/WholestageNativeRowRDD.scala

```diff
@@ -19,7 +19,7 @@ package com.intel.oap.execution
 
```

Comment:
Please change the class name to 'NativeWholestageRowRDD', and don't modify the file name, make the named form the same as 'NativeWholestageColumnarRDD'.

Reply:
Fixed.